src/components/KTreeList/KTreeDraggable.vue

```diff
@@ -222,11 +277,18 @@ const onStartDrag = (draggedItem: any): void => {
   setDragCursor(true)
 }
 
-const onStopDrag = (): void => {
+const onStopDrag = async (): Promise<void> => {
   dragging.value = false
   setDragCursor(false)
 
   key.value++
+
+  // set previous icon state after DOM was re-rendered
+  await nextTick(() => {
+    treeItems.value?.forEach((item: typeof KTreeItem) => {
+      item?.setExpandedValue(childrenVisibilityMap.value.get(item?.id))
+    })
+  })
```

Comment:
nit: can we update the syntax here to match the other usage in the repo?
```suggestion
  // set previous icon state after DOM was re-rendered
  await nextTick()
  treeItems.value?.forEach((item: typeof KTreeItem) => {
    item?.setExpandedValue(childrenVisibilityMap.value.get(item?.id))
  })
```